ros2/cc_defs.bzl

```diff
@@ -64,14 +64,15 @@ def ros2_c_binary(name, ros2_package_name = None, **kwargs):
     """
     _ros2_cc_target(cc_binary, "c", name, ros2_package_name, **kwargs)
 
-def _ros2_cpp_exec(target, name, ros2_package_name = None, set_up_ament = False, **kwargs):
-    if set_up_ament == False:
+def _ros2_cpp_exec(target, name, ros2_package_name = None, set_up_ament = False, set_up_ros_home = False, **kwargs):
```

Comment:
Why do we still need set_up_ros_home when the templates are updated?

Reply:
The idea was to keep setting up ament and setting up ROS_HOME orthogonal. We could change the flag to do both of course